packages/modelviewer.dev/examples/postprocessing/index.html

```diff
@@ -0,0 +1,140 @@
+      <div class="content">
+        <div class="wrapper">
+          <div class="heading">
+            <h2 class="demo-title">Toggle Effects</h2>
+            <p>This example demonstrates the use of PostProcessing Effects with the 
+            model-viewer. Enabling an effect is as simple as adding the attribute to 
+            the model-viewer element, or enabling it using the property. For example,
+            enabling a <a href="https://en.wikipedia.org/wiki/Bloom_(shader_effect)">Bloom Effect</a>
+            which is useful for emissive meshes can be done by adding the <code>bloom-effect</code>
+            attribute, or setting the <code>bloomEffect</code> attribute to <code>true</code>.
+            All implemented effects can be seen in this example, you can toggle them to see the difference.
+            <i>Note: the order of the effect attributes does not matter.</i></p>
+          </div>
+          <example-snippet stamp-to="effects" highlight-as="html">
+            <template>
+<model-viewer id="lantern" camera-controls touch-action="pan-y" src="../../shared-assets/models/Astronaut.glb" ar alt="A 3D model of a Lantern">
+  <mv-effects-composer>
+    <mv-pixelate-effect id="pixelate" granularity="30"></mv-pixelate-effect>
+  </mv-effects-composer>
+  <div class="controls">
+    <label for="granularity">Granularity</label>
+    <input id="granularity" type="range" min="10" max="100" step="0.5" value="30">
+  </div>
+</model-viewer>
+<script>
+const modelViewerPostprocessing = document.querySelector("model-viewer#lantern");
+const pixelate = modelViewerPostprocessing.querySelector('#pixelate');
+const granularity = modelViewerPostprocessing.querySelector('#granularity');
+
+function updateGranularity(event) {
+  pixelate.granularity = event.target.value;
+}
+
+granularity.addEventListener('change', updateGranularity);
+</script>
+            </template>
+          </example-snippet>
+        </div>
+      </div>
+    </div>
+
+    <div class="footer">
+      <ul>
+        <li class="attribution">
+          <a href="https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/Lantern">Lantern</a>,
+          licensed under <a href="https://creativecommons.org/publicdomain/zero/1.0/">Creative Commons Zero</a>.
+        </li>
+      </ul>
+      <div style="margin-top:24px;" class="copyright">©Copyright 2018-2020 Google Inc. Licensed under the Apache License 2.0.</div>
+      <div id='footer-links'></div>
+    </div>
+  </div>
+</div>
+
+  <script type="module" src="../../examples/built/docs-and-examples.js">
+  </script>
+  <script type="module">
+    (() => { init('examples-scenegraph'); })();
+    (() => { initFooterLinks();})();
+  </script>
+
+  <!-- Documentation-specific dependencies: -->
+  <script type="module"
+      src="../built/dependencies.js">
+  </script>
+
+  <!-- Loads <model-viewer> on modern browsers: -->
+  <script type="module"
+      src="../../node_modules/@google/model-viewer/dist/model-viewer.js">
```

Comment:
This example code is looking great from an API perspective. The next thing will be to import the library defining `<mv-effects-composer>` as its own `<script>` that is loaded in addition to `model-viewer.js`. That'll require another output from our rollup config. That'll be the key in making sure this is all feasible for us to ship. Do you feel unblocked there?

Reply:
I tried getting the separate library to link with packages/modelviewer.dev with rollup, it doesn't seem to be working yet so I'd like some help with that. Will make a push with the new library this week

Reply:
Sorry, been a bit busy/sick. Will try to take a look next week.